Configs/Module-Shared.xcconfig

```diff
@@ -228,4 +228,4 @@ TVOS_DEPLOYMENT_TARGET = 9.0
 VERSIONING_SYSTEM = apple-generic
 
 // Code will load on this and later versions of watchOS. Framework APIs that are unavailable in earlier versions will be weak-linked; your code should check for null function pointers or specific system versions before calling newer APIs.
-WATCHOS_DEPLOYMENT_TARGET = 2.0
+WATCHOS_DEPLOYMENT_TARGET = 3.0
```

Comment:
That's why we have `xcconfig` - to be able to change that setting in one place only

Reply:
This is ~~(the only one true way)~~ dirty hack in monolithic environment in xcode project. :) 
Xcode can't copy targets well, it also can't share build configurations and many more "monolithic" issues.